### PR TITLE
feat: juniper doc updates and liveaction parser

### PR DESCRIPTION
**Pull request by bparmar-splunk**
- Liveaction Parser
- Juniper default index doc update

### CODE REVIEW

File: package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+            r_set_splunk_dest_default(
+                index("netops")
+                sourcetype('liveaction:livenx')
+                vendor("liveaction")
+                product("livenx")
+            )
+        };
+    };
+};
+application app-syslog-liveaction_livenx[sc4s-syslog] {
```
Comment (rjha-splunk) [inline package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf:13]: looks like liveaction is a PROGRAM , can you check sc4s-syslog-pgm topic here, we have to filter based on program macro.

File: package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    filter {
+        message('%LIVEACTION'  type(string)  flags(prefix));
```
Comment (rjha-splunk) [inline package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf:15]: program('<string to use>' type(string) flags(prefix))

Comment (bparmar-splunk) [reply package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf:15]: Message is not considering `%LIVEACTION` as program and hence I have parsed it as message.

**Review by rjha-splunk — APPROVED** [review]

**Pull request merged**